app/components/sanctions/create-form.tsx

```
@@ -0,0 +1,515 @@
+    reviewStartsAt?.isValid && reviewValues
+      ? calculateSanctionEndsAt(
+          reviewStartsAt.toJSDate(),
+          reviewDuration,
+          reviewValues.validityUnit,
+        )
+      : null;
+  const reviewEndsAt = reviewEndsAtDate
+    ? DateTime.fromJSDate(reviewEndsAtDate).setZone(STORE_TIMEZONE)
+    : null;
+  const reviewStatus =
+    reviewStartsAt?.isValid && reviewStartsAt.toMillis() > Date.now()
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/app/components/sanctions/create-form.tsx:239:60
  237 |     : null;
  238 |   const reviewStatus =
> 239 |     reviewStartsAt?.isValid && reviewStartsAt.toMillis() > Date.now()
      |                                                            ^^^^^^^^^^ Cannot call impure function
  240 |       ? "scheduled"
  241 |       : "active";
  242 |

(react-hooks/purity)